openmrs-19/src/main/java/org/motechproject/openmrs19/domain/Encounter.java

```diff
@@ -18,6 +18,7 @@
     private Date encounterDatetime;
     private Patient patient;
     private Person provider;
+    private List<Person> encounterProviders;
```

Comment:
just name this field providers and please add comment here that this field is new in OpenMRS since version 1.10 (or 1.12 - check it).